src/redux/books/books.js

```diff
@@ -25,4 +30,60 @@ export const removeBook = (id) => ({
   id,
 });
 
+export const postBook = (payload) => (
+  async (dispatch) => {
+    await fetch(url, {
+      method: 'POST',
+      body: JSON.stringify({
+        item_id: payload.id,
+        title: {
+          bookTitle: payload.title,
+          bookAuthor: payload.author,
+        },
+        category: payload.category,
+      }),
+      headers: {
+        'Content-type': 'application/JSON',
+      },
+    });
+    dispatch(addBook(payload));
+  }
+);
```

Comment:
- [ ] Please take a step back and review this actions creator. Because it throws an error when you try to post your data to the API
- This is why after I create a book when I refresh the just created book does not apear
![Screenshot from 2022-03-31 21-18-03](https://user-images.githubusercontent.com/62705947/161132794-874a3801-842a-4864-9077-6c3c78cf7463.png)

Reply:
> * [ ]  Please take a step back and review this actions creator. Because it throws an error when you try to post your data to the API
> 
>     * This is why after I create a book when I refresh the just created book does not apear
>       ![Screenshot from 2022-03-31 21-18-03](https://user-images.githubusercontent.com/62705947/161132794-874a3801-842a-4864-9077-6c3c78cf7463.png)

Thank you @GzimAsani for reviewing my project. Everything is ok now.

Reply:
This issue is well resolved! ✔️